gateway/1.4/modules/ROOT/pages/flex-conn-tls-config.adoc

```diff
@@ -71,24 +73,24 @@ To apply a TLS context:
 . Navigate to either the:
 * Upstream or downstream configuration page, if adding a new API instance.
 * *Settings* page, if editing an existing API instance.
-. Select *HTTPS* for the *Protocol* configuration field if configuring an Inbound TLS context.
+. Select *HTTPS* for the *Protocol* configuration field if configuring an inbound TLS context.
 . Click *Add TLS Context*.
 .. Select a *Secret Group*.
 .. Select a *TLS Context*.
 .. Click *Ok*.
 . Finish creating the API instance or save the configuration edits.
 
 [[tls-context-applied-to-shared-ports]]
-== TLS Context Applied to Shared Ports
-Though a port must have an API Instance to have a TLS context applied, it is easiest to think of TLS context as being applied to the port rather than the API Instance. 
+== Inbound TLS Context Applied to Shared Ports
+Though a port must have an API instance to have an inbound TLS context applied, it is easiest to think of the inbound TLS context as being applied to the port rather than the API instance. 
```

Comment:
```suggestion
Though a port must have an API instance to have an inbound TLS context applied,  the inbound TLS context is applied to the port rather than the API instance. 
```

Reply:
Suggest simplifying the sentence. Not sure if this changes the meaning.